Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -878,6 +878,8 @@ namespace O3DE::ProjectManager
         }
         else
         {
+            //For Windows paths, need to normalize directory link to forward slashes
+            gemInfoResult.m_directoryLink.replace("\\", "/");
```

Comment:
Could run this through `AZ::IO::Path` normalize (`LexicallyNormal`)

Reply:
This can use the `AZ::IO::PathView` `AsPosix` function

Reply:
Eventually we should be storing all of our path type objects in AZ::IO::Path or similar types, instead of QStrings.  This has been on the backlog because it will touch a lot of code and will require careful testing.  I'm on the fence of doing a string replacement here with code that is specific to Windows - PALifying just this is a bit overkill, I'd rather run it through AZ::IO::Path.

Reply:
Addressed @AMZN-alexpete's suggestion in the latest commit via QDir.